crates/pg_workspace_new/src/workspace/server/document.rs

```diff
@@ -81,6 +81,31 @@ impl Document {
             .collect()
     }
 
+    pub fn line_and_col_to_offset(&self, line: u32, col: u32) -> TextSize {
```

Comment:
see prev comment - this should be done in the lsp layer, and respective helpers are provided in the `pg_lsp_converters` crate.